testing/test_pytest_html.py

```diff
@@ -238,9 +238,11 @@ def pytest_runtest_makereport(item, call):
 
     @pytest.mark.parametrize('content, encoded', [
         ("u'\u0081'", 'woE='),
-        ("'foo'", 'Zm9v')])
+        ("'foo'", 'Zm9v'),
+        (b"\xe2\x80\x93" if PY3 else "'\xe2\x80\x93'", "4oCT")])
```

Comment:
hmm, that test with the if/else looks wrong, since one encodes a quoted string , the other is merely a byte string, that seems wrong